src/csharp/Microsoft.Spark/Utils/CommandSerDe.cs

```diff
@@ -294,10 +342,20 @@ private static Delegate CreateUdfWrapperDelegate<T>(Type type, object[] paramete
                 parameters,
                 null);
 
-            return Delegate.CreateDelegate(
-                typeof(T),
-                udfWrapper,
-                UdfWrapperMethodName);
+            if (type == typeof(DataFrameGroupedMapUdfWrapper))
```

Comment:
Why is this needed? Why just this one type?

Reply:
Not needed anymore now that `DeserializeArrowOrDataFrameUdf` is passing in the right types. Deleted.